initfiles/componentfiles/configxml/espsmcservice.xsd.in

```diff
@@ -224,10 +224,10 @@
                 </xs:appinfo>
             </xs:annotation>
         </xs:attribute>
-        <xs:attribute name="elasticSearchAdresses" type="xs:string" use="optional" default="">
+        <xs:attribute name="elasticSearchAddresses" type="xs:string" use="optional" default="">
             <xs:annotation>
```

Comment:
@rpastrana @kunalaswani This attribute is referenced in esp/scm/ws_elk.ecm and  esp/services/ws_elk/WsELKService.cpp in the misspelled form